streams/standardization.js

```diff
@@ -0,0 +1,50 @@
+const Writable = require('stream').Writable;
+const Validator = require('jsonschema').Validator;
+const LOG_SCHEMAS = require('logs-schemas');
+
+
+class SchemaDebugStream extends Writable {
+    constructor(options) {
+        options = options || {};
+        super(options);
+        this.out = options.out || process.stderr;
+        delete options.out;
+
```

Comment:
I like the defense but is delete really needed?

Reply:
ohh nope, ha, the `super(options)` comes before 🙃